pkg/engine/templates_generated.go

```diff
@@ -12724,15 +12712,14 @@ if [[ ${OS} == "${UBUNTU_OS_NAME}" ]]; then
 fi
 DOCKER=/usr/bin/docker
 if [[ $UBUNTU_RELEASE == "18.04" ]]; then
-  export GPU_DV=450.80.02
+  export GPU_DV=450.51.06
```

Comment:
Did we mean to drop the version back to 450.51.06 from 450.80.02?

Reply:
The current deployment with the operator is `460.73.01`

Reply:
Not sure why they are using an older version here. The package is certainly newer. Best to open an issue upstream to get the version updated with the package (or have them split the drivers to a separate package).

Reply:
@RenaudWasTaken any idea why the nvidia-container-runtime 3.4 package would have older versions of the drivers than the 2.0.0 packages does?

Reply:
I see this we are actually downloading it in the provisioning script... most likely the function to trigger the download never ran and it had whatever version was baked into the VHD.

Reply:
Is the conclusion that we can use the newer version (450.80.02) with the v3 nvidia runtime?

Reply:
Note that the nVidia operator code was installing drivers with version 460.* (which seems like a big jump from 450.*)

Reply:
We should be able to the issue is I can't seem to make the download happen when deploying the cluster so it just has whatever is in the VHD.. and the versions end up not matching.

Reply:
@cpuguy83 I can get around that by using a vanilla 18.04 OS (not the VHD). I'll revert that driver downgrade and try.